sindresorhus/refined-github
# Add `friendlier-requested-changes` feature

Closes #1602 



# Test

- Go to a PR that has some requested changes from a review (i.e. https://github.com/sindresorhus/refined-github/pull/2667)
- Notice that the colour of the text and icons is now grey rather than red

Before:
<img width="750" alt="Screenshot 2019-12-09 22 23 58" src="https://user-images.githubusercontent.com/1215056/70474359-bc21af80-1ad2-11ea-8cb9-59e874ca15c2.png">

After:
<img alt="Screenshot 2019-12-09 22 22 43" src="https://user-images.githubusercontent.com/1215056/72876996-20ea3f80-3cf8-11ea-91a4-322b0fa25ecc.png">
